api/types.go

```diff
@@ -13,7 +14,7 @@ import (
 	"github.com/infrahq/infra/uid"
 )
 
-type Query map[string][]string
+type Query url.Values
```

Comment:
This is the same underlying type (`url.Values` is also a `map[string][]string`). I think it might help indicate how the `Query` type is used by declaring it as a `url.Values`.